sdk/contract_id.go

```diff
@@ -107,15 +107,16 @@ func ContractIDFromEvmAddress(shard uint64, realm uint64, evmAddress string) (Co
 // Does not populate ContractID.EvmAddress
 // Deprecated
 func ContractIDFromSolidityAddress(s string) (ContractID, error) {
```

Comment:
This is already marked as deprecated so I don't think we need to do anything here